core/domain/feedback_services.py

```diff
@@ -520,28 +400,18 @@ def _get_thread_from_model(thread_model):
     Returns:
         FeedbackThread. The corresponding FeedbackThread domain object.
     """
-    if constants.ENABLE_GENERALIZED_FEEDBACK_THREADS:
-        message_model = feedback_models.GeneralFeedbackMessageModel
-    else:
-        message_model = feedback_models.FeedbackMessageModel
     if thread_model.message_count is None:
```

Comment:
Why would this be None?

Reply:
I have filed an issue regarding message_counts being None (#5371). I'll investigate this as part of that issue.